static/js/map/map.pokemon.js

```diff
@@ -630,3 +648,17 @@ function sendPokemonNotification(pokemon) {
     notificationData.cp_multiplier = pokemon.cp_multiplier
     notifiedPokemonData[pokemon.encounter_id] = notificationData
 }
+
+function getSizeDisplay(size){
```

Comment:
Can you move this function to the utils.pokemon.js file?
Once it is there, please list it in the "exported" comment section at the top of the utils.pokemon.js file.
And, list it in the "globals" comment section at the top of the map.pokemon.js file.